src/main/resources/pom-servers.xml

```diff
@@ -623,6 +623,198 @@
 
         </profile>
 
+        <profile>
+            <id>kumuluzEE-1.3</id>
+
+            <properties>
+                <kumuluz.version>3.2.0</kumuluz.version>
```

Comment:
Should that not be version KumuluzEE version 2.6.0 which is MP 1.3 compliant.

Reply:
Updated